schema/proto3/auxiliarygateway.proto

```diff
@@ -0,0 +1,44 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+syntax = "proto3";
+
+package alcor.schema;
+
+option java_package = "com.futurewei.alcor.schema";
+option java_outer_classname = "AuxiliaryGateway";
+
+
+message AuxGateway {
+    // default value of empty string means no auxiliary gateway supported
+    string id = 1;
```

Comment:
This is a bit implicit. Maybe we could have two new fields
- IsEnable
- Gateway_Type

Reply:
added the below field to achieve both:
enum AuxGatewayType {
    NONE = 0;
    ZETA = 1;
}